src/cli/commands/test/iac-local-execution/file-scanner.ts

```diff
@@ -95,3 +97,18 @@ class PolicyEngine {
     }
   }
 }
+
+export class FailedToBuildPolicyEngine extends CustomError {
+  constructor(message?: string) {
+    super(message || 'Failed to build policy engine');
+    this.code = IaCErrorCodes.FailedToBuildPolicyEngine;
+    this.userMessage = '';
```

Comment:
I think later on when we have caching this could get more granular but for now we can go with.
```suggestion
    this.userMessage = 'We were unable run the test. Please run the command again with the `-d` flag and contact support@snyk.io with the contents of the output.';
```